lib/load-logos.js

```diff
@@ -2,13 +2,14 @@ var fs = require('fs');
 var path = require('path');
 
 var loadLogos = function() {
+  // Cache svg logos from disk in base64 string
   var logos = {};
-  var logoFiles = fs.readdirSync(path.join(__dirname, '..', 'logo'));
+  var logoDir = path.join(__dirname, '..', 'logo');
+  var logoFiles = fs.readdirSync(logoDir);
   logoFiles.forEach(function(filename) {
     if (filename[0] === '.') { return; }
     // filename is eg, github.svg
-    var svg = fs.readFileSync(
-      path.join(__dirname, '..', 'logo', filename)).toString();
+    var svg = fs.readFileSync(logoDir + '/' + filename).toString();
```

Comment:
Can you use a call to `path.join` here? It works better [across platforms](https://shapeshed.com/writing-cross-platform-node/#use-path-join-to-join-folder-names).